### ISSUE TITLE
[Feature Request - Web browser support] Add support for Vivaldi Snapshot channel

### ISSUE THREAD

**Issue by ugly95** [linuxmint/webapp-manager#56]
Vivaldi browser offers a snapshot channel in addition to the stable branch.  Only the stable branch is supported by webapp-manager.  The snapshot channel can be installed independently from the stable channel.

# 1 - Where did you get the browser from?

The Vivaldi snapshot channel is available from Vivaldi's official website: [https://vivaldi.com/blog/desktop/snapshots/](https://vivaldi.com/blog/desktop/snapshots/).  The snapshot release is initially installed using a .deb file.  Installation with the .deb file adds an 'Additional Repository' under 'Software Sources' (`/etc/apt/sources.list.d/vivaldi-snapshot.list` is created with `deb http://repo.vivaldi.com/snapshot/deb/ stable main` ).

As of January 12, 2020, the current snapshot is: [https://downloads.vivaldi.com/snapshot/vivaldi-snapshot_3.6.2160.3-1_amd64.deb](https://downloads.vivaldi.com/snapshot/vivaldi-snapshot_3.6.2160.3-1_amd64.deb)

# 2 - What is the absolute path to its binary?
```
whereis vivaldi-snapshot
vivaldi-snapshot: /usr/bin/vivaldi-snapshot
```
`/usr/bin/vivaldi-snapshot` is a symlink to `/opt/vivaldi-snapshot/vivaldi-snapshot`

**Comment by forsetius** [comment]
I already submitted a PR #54 :)

**Comment by musiclover382** [comment]
Fixed in #54. @ugly95 please close the issue
